CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8)
+
+include(GNUInstallDirs)
+
+find_package(LLVM REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+
+add_definitions(${LLVM_DEFINITIONS})
+include_directories(${LLVM_INCLUDE_DIRS})
+link_directories(${LLVM_LIBRARY_DIRS})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+add_definitions(-D_GNU_SOURCE
+	-DKLEE_DIR=\"${CMAKE_BINARY_DIR}\"
+	-DKLEE_INSTALL_BIN_DIR=\"${CMAKE_INSTALL_FULL_BINDIR}\"
+	-DKLEE_INSTALL_RUNTIME_DIR=\"${CMAKE_INSTALL_FULL_LIBDIR}/klee/runtime\")
+
+set(PACKAGE_STRING "\"KLEE 1.2.0\"")
+set(PACKAGE_URL "\"https://klee.github.io\"")
+set(HAVE_GPERFTOOLS_MALLOC_EXTENSION_H 1)
+set(ENABLE_STP 1)
+set(HAVE_MALLINFO 1)
+set(HAVE_MEMORY_H 1)
+set(KLEE_CMAKE_BUILD 1)
+set(RUNTIME_CONFIGURATION "\"\"")
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/include/klee/Config/config.h.cmin
```

Comment:
There are a bunch of checks missing here to check that the host platform supports certain things.